fmriprep/workflows/bold/t2s.py

```diff
@@ -76,6 +76,15 @@ def init_bold_t2s_wf(bold_echos, echo_times, mem_gb, omp_nthreads,
             the skull-stripped optimal combination mask
     """
     workflow = Workflow(name=name)
+    workflow.__desc__ = """\
+Voxel-wise T2* was estimated from preprocessed BOLD by fitting to a
```

Comment:
This is my preference, feel free to dismiss if you don't like it:

*Voxel-wise T2\** -> *A T2\* map*

Reply:
I see that you talk below about *For each voxel* (~voxel-wise) and you actually use *The T2\* map...*, which reinforces my preference of T2* map